make/Makefile.iso

```diff
@@ -38,7 +38,9 @@ $(MKSQUASHFS):
 	@echo "'mksquashfs' not found, install 'squashfs' package."
 	@exit 1
 
-create-repo: $(LUET) $(DESTINATION)
+create-repo: $(LUET) $(DESTINATION) $(DESTINATION)/tree.tar.zst
+
+$(DESTINATION)/tree.tar.zst:
```

Comment:
thing is, if we iterate locally and run multiple create-repos this won't work. It's not a one-off, but you could run create-repo more than one, maybe a `.PHONY` here?

Reply:
Hmm, I think a PHONY would do the same like reverting this diff.
If 'create-repo' is supposed to be run multiple times, let me change the dependencies of `local-iso` instead.